dart/lib/src/reference.dart

```diff
@@ -0,0 +1,416 @@
+import 'dart:collection';
+import 'dart:convert';
+import 'dart:typed_data';
+import 'types.dart';
+
+/// Main class to read a value out of a FlexBuffer.
```

Comment:
This doesn't really tell me much. The doc comments should say why I'd want to use this and what it does. I know it's a class, and I'm using the FlexBuffer library. Why do I want to use this class?